src/test/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGeneratorTest.java

```diff
@@ -94,21 +99,40 @@ void setup() {
     }
 
     @Test
-    void shouldGenerateCertificateOfService_whenValidDataIsProvided() {
+    void shouldGenerateCertificateOfService_whenValidDataIsProvided_forDefendant() {
```

Comment:
let's stick to `should_when` naming convention and add some details to it, maybe something like `shouldGenerateDefendantCertificateOfService_whenStateFlowIsFullDefence` (and similar for claimant `whenStateFlowIsNotFullDefence`